src/database.rs

```diff
@@ -48,6 +60,8 @@ impl<'a> Database<'a> {
 
     /// Execute a command on the database.
     /// This is performed lazily and therefore requires calling `next` on the resulting cursor.
+    /// Results are returned in batches as per the mongoc driver.
+    /// To get the next batch: https://docs.mongodb.com/manual/reference/command/getMore/
```

Comment:
I think you're meaning to say here that you cannot use the result if it contains batches? Maybe something like:

```
/// Execute a command on the database for small results that are not batched.
/// If you are expecting a result with more than X documents in it please use `command_batch`.
/// This is performed lazily and therefore requires calling `next` on the resulting cursor.
```